packages/compass-sidebar/src/components/sidebar-title.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useMemo } from 'react';
+
+import {
+  MongoDBLogoMark,
+  css,
+  cx,
+  uiColors,
+  spacing,
+  ItemActionControls,
+  useTheme,
+  Theme,
+} from '@mongodb-js/compass-components';
+
+import type { ItemAction } from '@mongodb-js/compass-components';
+
+type Actions =
+  | 'open-instance-workspace'
+  | 'copy-connection-string'
+  | 'edit-favorite'
+  | 'cluster-info';
+
+const titleLabel = css({
+  overflow: 'hidden',
+  whiteSpace: 'nowrap',
+  textOverflow: 'ellipsis',
+  fontSize: '18px', // TODO: what's the right way to do this?
```

Comment:
You probably can use one of the heading elements from leafygreen

Reply:
OK I'll try that in a separate PR - already building on this :)